application/org.openjdk.jmc.jolokia/build.properties

```diff
@@ -0,0 +1,45 @@
+#
+#  Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
```

Comment:
For new files, don't forget to add the right copyright year (2021), and a line for your current employer (if it makes sense). See Datadog copyrights.